HealthcareApp/src/Healthcare.Infrastructure/Repository/UserManagerDecorator.cs

```diff
@@ -8,13 +10,58 @@ namespace Infrastructure.Repository;
 public class UserManagerDecorator<TUser> : IUserManagerDecorator<TUser> where TUser : class
 {
     private readonly UserManager<TUser> _userManager;
+    private readonly AppDbContext _context;
 
-    public UserManagerDecorator(UserManager<TUser> userManager)
+
+    public UserManagerDecorator(UserManager<TUser> userManager, AppDbContext context)
     {
         _userManager = userManager;
+        _context = context;
     }
```

Comment:
Read about decorator pattern structure and write in tg why it's no longer a decorator. Fix pls